arm-sql/2014-04-01/swagger/serverDisasterRecoveryConfiguration.json

```diff
@@ -0,0 +1,430 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Sql Database API spec",
+    "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/disasterRecoveryConfiguration": {
```

Comment:
Talked to Johan.  This is normal behavior.  Will leave as is.

---
In reply to: [83288385](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r83288385) [](ancestors = 83288385,83281234)